### PR TITLE
Chore - sequelize setup

### PR DESCRIPTION
**Pull request by McHardex**
__What does this PR do?__
-  This PR sets up sequelize connections
__Description of Task to be completed?__
-  install `npm i sequelize-cli sequelize pg pg-hstore`
- setup sequelize configuration
- setup environment variables
- setup config file to manage node environments
- create postgres database
- connect sequelize to the database
__How should this be manually tested?__
- clone the repository
- git checkout to `ch-sequelize-setup` branch
- install dependencies `npm i`
- run `npm run start:conn` script to confirm connections to the appropriate database
__What are the relevant pivotal tracker stories?__

__Screenshots__

### CODE REVIEW

File: .env-example [hunk 1]
```diff
@@ -0,0 +1,7 @@
+DB_USERNAME=root
+DB_PASSWORD=password
+DEV_DATABASE=database_development
+PROD_DATABASE=database_production
+TEST_DATABASE=database_test
+DB_HOST=127.0.0.1
+DB_PORT=5432
```
Comment (sojida) [inline .env-example]: Please fix this EOLs

File: .sequelizerc [hunk 2]
```diff
@@ -0,0 +1,8 @@
+const path = require('path');
+
+module.exports = {
+    'config': path.resolve('./server/config', 'config.js'),
+    'models-path': path.resolve('./server/models'),
+    'seeders-path': path.resolve('./server/seeders'),
+    'migrations-path': path.resolve('./server/migrations')
+}
```
Comment (sojida) [inline .sequelizerc]: EOLs

File: package.json [hunk 3]
```diff
@@ -6,8 +6,10 @@
   "scripts": {
     "start": "npm run build && node dist/app.js",
     "build": "babel server/src -d dist",
-    "start:dev": "nodemon server/src/app.js --exec babel-node",
-    "test": "export NODE_ENV=test || SET \"NODE_ENV=test\" && nyc mocha test/**/*.test.js --timeout 20000 --require @babel/register --exit"
+    "start:dev": "export NODE_ENV=development && nodemon server/src/app.js --exec babel-node",
+    "start:conn": "babel-node server/models",
+    "pretest": "export NODE_ENV=test || SET \"NODE_ENV=test\" && npm run start:conn",
```
Comment (vic3king) [inline package.json]: Our Pretest is back! Please, where is the start:conn script?

**Comment by sojida** [comment]
Great work @McHardex  and @walsamlee.

File: server/config/config.js [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+const dialect = 'postgres';
+
+module.exports = {
+  development: {
+    username: process.env.DB_USERNAME,
+    password: process.env.DB_PASSWORD,
+    database: process.env.DEV_DATABASE,
+    host: process.env.DB_HOST,
+    port: process.env.DB_PORT,
+    dialect,
```
Comment (amaechi-chuks) [inline server/config/config.js]: You can as well add operatorsAliases: false, after the end of this line to solve sequelize depreciation issues.

**Pull request merged**